pkg/storage/etcd3/watcher_test.go

```diff
@@ -227,14 +265,17 @@ func (c *testCodec) Decode(data []byte, defaults *unversioned.GroupVersionKind,
 	return nil, nil, errors.New("Expected decoding failure")
 }
 
-func testCheckResult(t *testing.T, i int, expectEventType watch.EventType, w watch.Interface, expectObj *api.Pod) {
+func testCheckResult(t *testing.T, i int, expectEventType watch.EventType, w watch.Interface, expectObj *api.Pod, ignoreResult bool) {
```

Comment:
actually... how about adding another function called testExpectReceivingEvent, which only tries to receive event from the given chan? I dislike the bool arg.